app/scripts/controllers/preferences.js

```diff
@@ -81,7 +76,6 @@ export default class PreferencesController {
     this.store = new ObservableStore(initState);
     this.store.setMaxListeners(12);
     this.openPopup = opts.openPopup;
-    this.migrateAddressBookState = opts.migrateAddressBookState;
```

Comment:
It looks like this option is no longer used, but it's still being passed in during construction. This option can now be removed in `metamask-controller.js`, and the function itself can be deleted.

Reply:
done [here](https://github.com/MetaMask/metamask-extension/pull/17127/commits/7dd9f4b92f9c1ae722b5c65978e0b6edc5100325)